test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -626,5 +632,21 @@ def main():
     if trapped_exceptions:
         sys.exit(1)
 
+# Remove the lowest bit from the core dump mask. The lowest bit is for
+# anonymous private mappings (see: `man core`). These mappings are used by
+# ASAN, MSAN et al to allocate shadow memory. Some of these mappings are
+# correctly marked as "don't dump" via `memadvise`, but not
+# all. Unfortunately these mappings are quite large (multiple terabytes), so
+# don't write them to disk.
+#
+# In theory there could be WT-internal use cases for these mappings, but as
```

Comment:
I would just say WiredTiger instead of WT-internal here but again not a big deal

Reply:
fixed in 49ee77f